2021-01-12 20:15:57,781 INFO    MainThread:8976 [wandb_setup.py:_flush():70] setting env: {}
2021-01-12 20:15:57,782 INFO    MainThread:8976 [wandb_setup.py:_flush():70] multiprocessing start_methods=spawn
2021-01-12 20:15:57,782 INFO    MainThread:8976 [wandb_setup.py:_flush():70] setting login settings: {}
2021-01-12 20:15:57,782 INFO    MainThread:8976 [wandb_init.py:_log_setup():303] Logging user logs to C:\Users\kevin\OneDrive\Documents\8th sem\Federated_Pneumonia\src\aggregator\wandb\run-20210112_201557-1t9fegxm\logs\debug.log
2021-01-12 20:15:57,782 INFO    MainThread:8976 [wandb_init.py:_log_setup():304] Logging internal logs to C:\Users\kevin\OneDrive\Documents\8th sem\Federated_Pneumonia\src\aggregator\wandb\run-20210112_201557-1t9fegxm\logs\debug-internal.log
2021-01-12 20:16:00,119 INFO    MainThread:8976 [wandb_run.py:_console_start():1237] atexit reg
2021-01-12 20:16:00,120 INFO    MainThread:8976 [wandb_run.py:_redirect():1104] redirect: SettingsConsole.WRAP
2021-01-12 20:16:00,120 INFO    MainThread:8976 [wandb_run.py:_redirect():1135] Wrapping output streams.
2021-01-12 20:16:00,120 INFO    MainThread:8976 [wandb_run.py:_redirect():1151] Redirects installed.
2021-01-12 20:24:00,324 INFO    MainThread:8976 [wandb_run.py:_atexit_cleanup():1207] got exitcode: 0
2021-01-12 20:24:00,325 INFO    MainThread:8976 [wandb_run.py:_restore():1179] restore
2021-01-12 20:24:00,353 INFO    MainThread:8976 [wandb_run.py:_wait_for_finish():1324] got exit ret: uuid: "5e9b6d03c78144009ba329d34614024f"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
    }
    pusher_stats {
      uploaded_bytes: 984
      total_bytes: 984
    }
  }
}

2021-01-12 20:24:02,367 INFO    MainThread:8976 [wandb_run.py:_wait_for_finish():1324] got exit ret: uuid: "ee66cfacdd664dd69ea2594a4e20b3f7"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 9393
      total_bytes: 9393
    }
  }
}

2021-01-12 20:24:04,371 INFO    MainThread:8976 [wandb_run.py:_wait_for_finish():1324] got exit ret: uuid: "a922caf14c1f4ea3aa9a70696872eddf"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 9393
      total_bytes: 9393
    }
  }
}

2021-01-12 20:24:05,572 INFO    MainThread:8976 [wandb_run.py:_show_summary():1469] rendering summary
2021-01-12 20:24:05,578 INFO    MainThread:8976 [wandb_run.py:_show_history():1507] rendering history
2021-01-12 20:24:05,594 INFO    MainThread:8976 [wandb_run.py:_show_files():1536] logging synced files
2021-01-12 20:24:05,752 INFO    MainThread:8976 [internal.py:handle_exit():138] Internal process exited
